src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -341,9 +341,8 @@ def load_arguments(self, _):
         c.argument('action_group_name', options_list=['--name', '-n'], id_part='name')
 
     with self.argument_context('monitor action-group create') as c:
-        from .validators import process_action_group_detail_for_creation
         from .actions import ActionGroupReceiverParameterAction
-        c.extra('receivers', options_list=['--action', '-a'], nargs='+', arg_group='Actions', action=ActionGroupReceiverParameterAction, validator=process_action_group_detail_for_creation)
+        c.extra('receivers', options_list=['--action', '-a'], nargs='+', arg_group='Actions', action=ActionGroupReceiverParameterAction)
```

Comment:
Move it to `g.command('create', 'create_or_update', table_transformer=action_group_list_table, validator=process_action_group_detail_for_creation)` now, because :
- The validator validates not only `receivers`, but also other arguments.
- I need use command validator (`process_action_group_detail_for_creation`) to overwrite argument validator (`validate_tags`) as`validate_tags` should not used for this command as `tags` should be in payload.

---
In reply to: [559287114](https://github.com/Azure/azure-cli/pull/16551#discussion_r559287114) [](ancestors = 559287114)